packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -176,58 +179,48 @@ export const ErrorField = styled.p`
 
 const Row = styled.div`
   display: flex;
-  flex: 1;
-  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
-    flex-direction: column;
+  gap: 24px;
+  width: 100%;
+  justify-content: center;
+  background-color: ${({ theme }) => theme.colors.background};
+  flex-direction: row;
+  padding: 24px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
+    padding: 0;
   }
 `
+const Wrapper = styled.div``
```

Comment:
this wrapper needs some styling here. As previously the review section had FullBodyContent as container, and it centered the whole thing in the center of the page. Not some of the top part of the reviewsection get beneath the eventTopBar. This div needs some styles for margin